src/Raven.Server/Routing/RequestRouter.cs

```diff
@@ -405,7 +406,7 @@ await using (var writer = new AsyncBlittableJsonTextWriter(ctx, context.Response
                 feature.Status == RavenServer.AuthenticationStatus.NoCertificateProvided)
             {
                 message = "This server requires client certificate for authentication, but none was provided by the client. Did you forget to install the certificate?";
-                message += BrowserCertificateMessage;
+                message += context.Request.IsFromStudio() ? BrowserCertificateMessage : string.Empty;
```

Comment:
This is more complex. Probably needs to be based on User-Agent or Raven-Client-Version (lack of it)

Reply:
done